examples/lib/intermediateHtmlStepUi.js

```diff
@@ -65,6 +67,7 @@ function IntermediateHtmlStepUi(_sequencer, step, options) {
       .insertAdjacentElement('afterend',
         addStepUI
       );
+      updatePreviews(_sequencer.images.image1.steps[0].options.step.imgElement.src,'insertStep');
```

Comment:
Wow this is long! Does insertStep not have access to the current step somehow simpler?

Reply:
Please review.